static/app/views/performance/browser/webVitals/pageOverview.tsx

```diff
@@ -93,6 +95,17 @@ export default function PageOverview() {
     return null;
   }
 
+  const transactionSummaryTarget =
+    project &&
+    !Array.isArray(location.query.project) && // Only navigate to transaction summary when one project is selected.
```

Comment:
so the button doesn't do anything if no projects are selected? Should conditionally display a link if so?

Reply:
The comment could use an update though, I think thats where the confusion is.

Reply:
It doesn't get rendered at all for multiple projects